### PR TITLE
Package dog.0.0.4

### PR DESCRIPTION
**Pull request by muqiuhan**
### `dog.0.0.4`
A simple OCaml logging library
A simple OCaml logging library

---
* Homepage: https://github.com/muqiuhan/dog
* Source repo: git+https://github.com/muqiuhan/dog.git
* Bug tracker: https://github.com/muqiuhan/dog/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/dog/dog.0.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+license: "MIT"
+tags: ["log" "logging" "logger"]
+homepage: "https://github.com/muqiuhan/dog"
+doc: "https://github.com/muqiuhan/dog"
+bug-reports: "https://github.com/muqiuhan/dog/issues"
+depends: [
+  "ocaml" {>= "4.11.0"}
+  "dune" {>= "3.5"}
+  "ocolor"
+  "core_kernel"
```
Comment (avsm) [inline packages/dog/dog.0.0.4/opam:16]:
```suggestion
  "core_kernel" {>="v0.16.0"}
```

fixes lower bounds failure

**Comment by avsm** [comment]
There's already a package called `dog` in the opam repository (see https://ocaml.org/p/dog/latest by @samoht).  Would you mind renaming your package to something else? You can just rename the directory of the opam package here in this PR; no need to change your own project.

**Comment by muqiuhan** [comment]
sorry, I have released [simlog #23994 ](https://github.com/ocaml/opam-repository/pull/23994) in opam before. This project is simlog (renamed dog in 0.0.4). I will rename this package to simlog and submit a new [pr #24628 ](https://github.com/ocaml/opam-repository/pull/24628).

**Comment by avsm** [comment]
Thanks!

**Pull request closed without merging**